README.md

```diff
@@ -152,6 +163,24 @@ VALID_ISSUERS=
 
 Once the self service app is started, the aggregator api will work as well
 
+## NPM Commands
```

Comment:
Moved this below the deployment steps since I somehow feel that those are more important than this. The list of commands can also be seen from package.json file.